binaryaudit/abicheck.py

```diff
@@ -253,8 +253,8 @@ def filter_dictionary(rpm_dict, drop_count):
                     kernel_has_debuginfo = True
                     break
             if kernel_has_debuginfo is False:
-                util.note("Dropping files with " + key +
-                          " source name because kernel packages must be accompanied by debuginfo package")
+                util.note("Dropping files with {} source name because ".format(key)
+                          + "kernel packages must be accompanied by debuginfo package")
```

Comment:
This would be like below, i'd guess:

```python
util.note("Dropping files with {} source name because " \
          "kernel packages must be accompanied by debuginfo package".format(key))
```